### ISSUE TITLE
Track Location Changes Over Time

### ISSUE THREAD

**Issue by b-camphart** [Soyle-Productions/soyle-stories-desktop#111]
# 🚀 Feature Request

### 📝 Description

A city may crumble into dust, a lake my dry up, a country's borders may expand, the architecture of a town may evolve.

### ✨ Describe the solution you'd like

  
We should allow users to add an image to a location to use as a map and allow that image to be replaced during various story events.  It could work similarly to animation software in that each story event with a new image functions as a "keyframe" and any story event that takes place at that location, but makes no changes, functions as a "frame."  This would allow new story events that take place at a location after a particular change to the location to immediately show the correct state of that location.

This should also be used for the description of the location as well.  Some writers may be more visual and want to draw out their locations, but some may not have that ability and just want to describe it instead.  Many will do both.

**Comment by b-camphart** [comment]
This feature depends on #105 being completed first.
